packages/front-end/components/Layout/Layout.tsx

```diff
@@ -342,6 +359,21 @@ const Layout = (): React.ReactElement => {
             View Docs <FaArrowRight className="ml-2" />
           </a>
         </div>
+        {!isCloud() && isOldVersion && (
+          <div className="px-3 my-1 text-center">
+            <small>
+              <span className="text-muted">GrowthBook has a new version:</span>{" "}
+              <a
+                href={`https://docs.growthbook.io/self-host/updating`}
+                target="_blank"
+                rel="noreferrer"
+                className="text-white"
+              >
+                {`Updating to Latest Docs`}
```

Comment:
I think this wording is a little confusing. Maybe "View Update Instructions" would be clearer.